### PR TITLE
Added unit tests for both new features.

### PR DESCRIPTION
**Pull request by sentient06**

### CODE REVIEW

File: test/week_test.js [hunk 1]
```diff
@@ -57,6 +57,44 @@ describe('Week', () => {
...
+
+    const weekStart = moment('2015-12-20')
+    const realWeekNumber = parseInt(weekStart.format('w'), 10)
+    const week = mount(
+      <Week day={weekStart} showWeekNumber onWeekSelect={onWeekClick} />
+    )
+    const weekNumberElement = week.find(WeekNumber)
+    weekNumberElement.simulate('click')
+    expect(firstDayReceived).to.equal(weekStart.format())
+    expect(weekNumberReceived).to.equal(realWeekNumber)
```
Comment (duncanfwalker) [inline test/week_test.js]: this two expectations are orthogonal so should be in separate tests

(this applies to the 'should set the week number with the provided formatWeekNumber function' test too)

File: test/week_test.js [hunk 2]
```diff
@@ -57,6 +57,44 @@ describe('Week', () => {
     assert(day.prop('day').isSame(dayClicked, 'day'))
   })
 
+  it('should call the provided onWeekSelect function', () => {
+    let firstDayReceived = null
+    let weekNumberReceived = null
+
+    function onWeekClick (day, newWeekNumber) {
```
Comment (duncanfwalker) [inline test/week_test.js]: this two expectations are orthogonal so should be in separate tests

(this applies to the 'should set the week number with the provided formatWeekNumber function' test too)

File: test/week_test.js [hunk 3]
```diff
@@ -57,6 +57,44 @@ describe('Week', () => {
...
+      firstDayReceived = day.format()
+      weekNumberReceived = newWeekNumber
+    }
+
+    const weekStart = moment('2015-12-20')
+    const realWeekNumber = parseInt(weekStart.format('w'), 10)
+    const week = mount(
```
Comment (duncanfwalker) [inline test/week_test.js]: shouldn't need to do a full mount, shallow better isolates the system under test

Comment (sentient06) [reply test/week_test.js]: Doesn't work with shallow.

File: test/week_test.js [hunk 4]
```diff
@@ -57,6 +57,57 @@ describe('Week', () => {
...
+      return 9
+    }
+
+    const weekStart = moment('2015-12-20')
+    const week = shallow(
+      <Week day={weekStart} showWeekNumber formatWeekNumber={weekNumberFormatter} />
+    )
+    const weekNumberElement = week.find(WeekNumber)
+
+    expect(firstDayReceived).to.equal(weekStart.format())
```
Comment (duncanfwalker) [inline test/week_test.js]: very minor point but..
when i first read this it took me a while to work out why you were using .format(), here and on L97. I think you are doing it get a string so that value comparison works in the expectation.

I think a more intention revealing way to do this would be to use moments isSame() eg:
expect(recievedDay.isSame(weekStart)).to.be.true

**Review by duncanfwalker — APPROVED** [review]
